tests/tests_crypto_connect.py

```diff
@@ -25,28 +25,28 @@ class TestCryptoConnect(unittest.TestCase):
 
     '''
     @classmethod
-    def setUpClass(cls):
+    def setUpClass(self):
         # start our own server
-        cls.srv_crypto = Server()
```

Comment:
calling the class cls is convention in classmethod. not an error. This is should have been left as cls

Reply:
Sorry about that! Good to know.

Reply:
Should I change my patch, or generate a new pull request to fix this?

Reply:
if you could generate a new PR it would be great. Thanks for your work! cryptography is definitely a part of that library that need attention